treeherder/webapp/api/jobs.py

```diff
@@ -171,16 +171,6 @@ class JobsViewSet(viewsets.ViewSet):
     _option_collection_hash_idx = [pq[0] for pq in _property_query_mapping].index(
         'option_collection_hash')
 
-    def _get_option_collection_map(self):
```

Comment:
I've taken care of this in this commit:
https://github.com/mozilla/treeherder/pull/1957/commits/5cf1917ccdff68a8db5a280f3fe0d51d22133045

I've also modified refdata.py to use the new method.